src/plugins/intel_gpu/tests/fusions/deconvolution_fusion_test.cpp

```diff
@@ -596,81 +564,81 @@ TEST_P(deconv_scale_actv_quant_i8, basic) {
 }
 
 INSTANTIATE_TEST_SUITE_P(fusings_gpu, deconv_scale_actv_quant_i8, ::testing::ValuesIn(std::vector<deconv_test_params>{
-    deconv_test_params{ CASE_DECONV_FP32_1, 2, 2, 5 },
-    deconv_test_params{ CASE_DECONV_FP32_2, 2, 2, 5 },
-    deconv_test_params{ CASE_DECONV_FP32_3, 2, 2, 5 },
-    deconv_test_params{ CASE_DECONV_FP32_4, 2, 2, 5 },
-    deconv_test_params{ CASE_DECONV_FP32_5, 2, 2, 5 },
-    deconv_test_params{ CASE_DECONV_FP32_6, 2, 2, 5 },
-    deconv_test_params{ CASE_DECONV_FP32_7, 2, 2, 5 },
-    deconv_test_params{ CASE_DECONV_FP32_8, 2, 2, 5 },
-
-    deconv_test_params{ CASE_DECONV_FP16_1, 2, 2, 5 },
-    deconv_test_params{ CASE_DECONV_FP16_2, 2, 2, 5 },
-    deconv_test_params{ CASE_DECONV_FP16_3, 2, 2, 5 },
-    deconv_test_params{ CASE_DECONV_FP16_4, 2, 2, 5 },
-    deconv_test_params{ CASE_DECONV_FP16_5, 2, 2, 5 },
-    deconv_test_params{ CASE_DECONV_FP16_6, 2, 2, 5 },
-    deconv_test_params{ CASE_DECONV_FP16_7, 2, 2, 5 },
-    deconv_test_params{ CASE_DECONV_FP16_8, 2, 2, 5 },
-
-    deconv_test_params{ CASE_DECONV_U8S8_1, 2, 2, 5 },
-    deconv_test_params{ CASE_DECONV_U8S8_2, 2, 2, 5 },
-    deconv_test_params{ CASE_DECONV_U8S8_3, 2, 2, 5 },
+    deconv_test_params{ CASE_DECONV_FP32_1, 2, 3, 5 },
```

Comment:
Please leave an explanation in commit message.

Reply:
Okay. I'll try to write commit message more detailed.